superset/assets/src/SqlLab/components/AceEditorWrapper.jsx

```diff
@@ -27,6 +27,11 @@ import { areArraysShallowEqual } from '../../reduxUtils';
 
 const langTools = ace.acequire('ace/ext/language_tools');
 
+const SQL_KEYWORD_AUTOCOMPLETE_SCORE = 100;
```

Comment:
where are those `score: XXXXX_ SCORE` used?

Reply:
They're used by ace editor's autocomplete logic to determine the ordering of autocomplete results